src/fixate/drivers/ftdi.py

```diff
@@ -54,6 +57,19 @@ class FT_STATUS(object):
     FT_OTHER_ERROR = DWORD(18)
 
 
+def get_status_message(value):
+    """
+    :param value: Integer representing the error message
+    :return:
+    """
+    for msg, itm in FT_STATUS.__dict__.items():
+        try:
+            if itm.value == value:
+                return msg
+        except AttributeError:
+            pass
```

Comment:
This either needs a comment to justify why it's O.K. to ignore the exception, or improved in some other way.

Assuming that msg is a string, returning a empty string or something like f"No error message found for error number '{value}'"?

Reply:
Changed FT_STATUS to a dictionary as it isn't used outside of error checking. (ie. the DWORD ctypes are never needed or used.)
```python
class FTD2XXError(BaseException):
    pass


def check_return(return_code):
    if return_code != 0:
        raise FTD2XXError(FT_STATUS.get(return_code, "FT_UNKNOWN_ERROR"))
```